instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import CommentSection from '../CommentSection/CommentSection';
+
+
+const PostContainer = props => {
+    return (
+        <div>
+            {props.dumData.map(usersdata => (
+              <CommentSection key={Math.random(Date.now())} usersdata={usersdata} />  
```

Comment:
Using Math.random and Date.now is great way to randomize id's for such a small project. We will learn more complex ways once we get to the security portion of the curriculum at Lambda. Great job!